### PR TITLE
Allison

### PR DESCRIPTION
**Pull request by adskinner**
Hi Jordan! I've wrapped up my slides/images. You can take a look & merge into repo w yours. Let me know if you need anything else from me!

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -40,6 +40,197 @@
           <h1>Web Fundamentals: Web Systems</h1>
           <h2>Four Athens</h2>
         </section>
+          <section>
+              <div align="center">
```
Comment (PendragonDevelopment) [inline public/index.html]: It's not a requirement, but adding `class="no-border"` to the images will supersede the need for the div here.

File: public/index.html [hunk 2]
```diff
@@ -40,6 +40,197 @@
...
+           <section>
+          <h1>What is a 'Call To Action'? (CTA)</h1>
+               <br>
+               <p>What <em>action</em> do you want your visitors to your website to make?</p>
+        </section>
+           <section>
+          <h2>CTA Examples</h2>
+               <br>
+               <ul>
+                   <li>user fills out contact form</li>
```
Comment (PendragonDevelopment) [inline public/index.html]: Again, not a requirement, but adding `class="fragment"` to the `<li>`s will let them roll in step by step.

**Comment by adskinner** [comment]
done! thanks

**Comment by PendragonDevelopment** [comment]
Yeeeeeeah!

**Pull request merged**